samples/extensions/mesh_shader_culling/README.md

```diff
@@ -0,0 +1,198 @@
+<!--
+- Copyright (c) 2023, Holochip Corporation
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+-->
+
+# Mesh Shader Culling
+![Mesh Shader Culling](./images/mesh_shader_culling.png)
+
+## Overview
+
+This sample demonstrates how to incorporate the Vulkan extension [```VK_EXT_mesh_shader```](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VK_EXT_mesh_shader), and
+introduces mesh shader culling using meshlets.
```

Comment:
The term "mesh shader culling" is vague, it should either be:

* per-meshlet culling in a task shader, or
* per-primitive culling in a mesh shader

Reply:
(or both)